File: recommend/schemas/cooler.py
from pydantic import BaseModel
from typing import Optional

class CoolerSchema(BaseModel):
    id: int
    name: str
    price: Optional[int] = None
    link: str
    company: str
    product_seq: int
    image: Optional[str] = None
    category: Optional[str] = None
    cooling_type: Optional[int] = None
    aircool_form: Optional[int] = None
    tdp: Optional[int] = None
    intel_socket: Optional[int] = None
    amd_socket: Optional[int] = None
    fan_size: Optional[int] = None
    fan_count: Optional[int] = None
    airflow: Optional[int] = None
    noise: Optional[float] = None
    width: Optional[float] = None
    length: Optional[float] = None
    height: Optional[float] = None
    radiator: Optional[int] = None
    radiator_length: Optional[float] = None
    radiator_thickness: Optional[float] = None
    hose_length: Optional[float] = None
    feature: Optional[int] = None
    as_years: Optional[int] = None
    reg_date: Optional[int] = None
    bookmark: Optional[int] = 0

def serialize_cooler(cooler_object):
    cooler_dict = CoolerSchema(
        id=cooler_object.id,
        name=cooler_object.name,
        price=cooler_object.price,
        link=cooler_object.link,
        company=cooler_object.company,
        product_seq=cooler_object.product_seq,
        image=cooler_object.image,
        category=cooler_object.category,
        cooling_type=cooler_object.cooling_type,
        aircool_form=cooler_object.aircool_form,
        tdp=cooler_object.tdp,
        intel_socket=cooler_object.intel_socket,
        amd_socket=cooler_object.amd_socket,
        fan_size=cooler_object.fan_size,
        fan_count=cooler_object.fan_count,
        airflow=cooler_object.airflow,
        noise=cooler_object.noise,
        width=cooler_object.width,
        length=cooler_object.length,
        height=cooler_object.height,
        radiator=cooler_object.radiator,
        radiator_length=cooler_object.radiator_length,
        radiator_thickness=cooler_object.radiator_thickness,
        hose_length=cooler_object.hose_length,
        feature=cooler_object.feature,
        as_years=cooler_object.as_years,
        reg_date=cooler_object.reg_date,
        bookmark=cooler_object.bookmark
    ).__dict__
    return cooler_dict

File: backend/src/main/java/com/windows33/cavendish/domain/part/cooler/entity/Cooler.java
package com.windows33.cavendish.domain.part.cooler.entity;

import com.windows33.cavendish.global.entity.PartCommonEntity;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import javax.persistence.*;

@Getter
@SuperBuilder
@NoArgsConstructor
@Table(name = "cooler")
@Entity
public class Cooler extends PartCommonEntity {

    private String category;

    private Integer coolingType;

    private Integer aircoolForm;

    private Integer tdp;

    private Integer intelSocket;

    private Integer amdSocket;

    private Integer fanSize;

    private Integer fanCount;

    private Integer airflow;

    private Float noise;

    private Float width;

    private Float length;

    private Float height;

    private Integer radiator;

    private Float radiatorLength;

    private Float radiatorTickness;

    private Float hoseLength;

    private Integer feature;

    private Integer asYears;

    private Double radiatorThickness;

}